src/LinAlg/SAM.C

```diff
@@ -944,3 +944,25 @@ bool SAM::getNodalReactions (int inod, const Vector& rf, Vector& nrf) const
 
   return haveRF;
 }
+
+
+IntSet SAM::getEquations(char dofType, int dof) const
```

Comment:
1) Ligningsnr. _er_ unike av natur. Hvis to ulike frie DOF's hadde felles ligningsnr. er det noe veldig rart et sted og det vil nok da gå i knas andre steder om det skulle være tilfelle. Derfor er det ikke nødvendig å sjekke for uniqe-ness her (slik som set implisitt gjør).
2) Dette er da eneste (og god nok) grunn.